docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
+  :ref:`unrestricted components <component-restricted>`.
 
-.. describe:: VCS
+- The restrictions imposed by group's :guilabel:`Languages` are applied later,
+  when the access to perform certain action is checked, and it only affects just
+  some translation-related permissions.
 
-    Can manage VCS and access the exported repository.
+.. hint::
 
-.. describe:: Administration
+   Use :guilabel:`Language selection` or :guilabel:`Project selection`
+   to automate inclusion of all languages or projects.
 
-    Has all permissions available in the project.
+**Example:**
 
-.. describe:: Billing
+  Let's say there is a project ``foo`` with two component: ``foo/bar`` and
+  ``foo/baz`` and the next group:
 
-    Can access billing info (see :ref:`billing`).
+  .. list-table:: Group `Spanish Admin-Reviewers`
+         :stub-columns: 1
 
+         * - Roles
+           - `Review Strings`, `Manage repository`
+         * - Components
+           - foo/bar
+         * - Languages
+           - `Spanish`
 
-.. _custom-acl:
+..
 
-Custom access control
----------------------
+  Members of that group will have next permissions (assuming the default role settings):
 
-To gain more access control adjustments in a project, you can set
-:guilabel:`Access control` to :guilabel:`Custom` to switch over to
-using the Django admin-interface instead of the one in Weblate.
+    - General access (for browsing) to the whole project ``foo`` including both
+      its subcomponents: ``foo/bar`` and ``foo/baz``.
+    - Review strings in ``foo/bar`` Spanish translation (and only it).
+    - Manage VCS for the whole ``foo/bar`` repository e.g. commit pending
+      changes made by translators for all languages.
 
-If you want to do this by default for all current and new projects, configure the
-:setting:`DEFAULT_ACCESS_CONTROL` to administrate all permissions and relations using
-the Django admin interface.
+.. _autogroup:
 
-.. warning::
+Automatic group assignments
++++++++++++++++++++++++++++
 
-    By turning this on, Weblate will remove all :ref:`acl` it has created for
-    this project. If you are doing this without admin permission from the instance, you
-    will instantly lose your access to manage the project.
+On the bottom of :guilabel:`Group` editing page in the
+:ref:`Django admin interface <admin-interface>` you can specify
+:guilabel:`Automatic group assignments`, which is a list of regular expressions
+used to automatically assign newly created users to the group based on their
+e-mail addresses. This assignment only happens upon account creation.
 
-.. _default-groups:
+The most common use case for the feature is to assign all new users to some
+default group. In order to do so you will probably want to keep the default
+value (``^.*$``) for the regexp. Another use case for this option might be to
+give some additional privileges to employees of your company by default.
+Assuming all of them use corporative e-mail addresses on your domain, this can
```

Comment:
```suggestion
Assuming all of them use corporate e-mail addresses on your domain, this can
```
Is it @yourdomain or is it @corporate1 and @corporate2 on @yourdomain?